app/workers/application_worker.rb

```diff
@@ -0,0 +1,50 @@
+require 'prx_access'
+
+class ApplicationWorker
+  include PrxAccess
+  include Shoryuken::Worker
+
+  attr_accessor :message
+
+  def self.prefix_name(name)
+    [prefix, application, name].join('_')
+  end
+
+  def self.announce_queues(model, actions)
+    actions.map do |action|
+      [prefix, 'announce', application, model, action].join('_')
+    end
+  end
+
+  def self.application
+    'feeder'
+  end
+
+  def self.prefix
+    Rails.configuration.active_job.queue_name_prefix
+  end
+
+  def announce_perform(event)
```

Comment:
Fancy!  But I suppose this is very similar to what Announce did under the hood anyways.

Reply:
essentially I reimplemented a "light" version announce, or just the parts we need and no more ;)